knowledge graph/kg_agent.py

```
@@ -0,0 +1,356 @@
+import asyncio
+import json
+from langchain.agents import AgentExecutor
+from langchain.agents.agent_types import AgentType
```

Ruff (0.12.2)

`langchain.agents.agent_types.AgentType` imported but unused

Remove unused import: `langchain.agents.agent_types.AgentType`

(F401)